ember-cli-build.js

```diff
@@ -271,8 +271,8 @@ module.exports = function() {
   ];
 
   if (hasBower) {
```

Comment:
Since you moved `loader.js` from bower, this check seems to not make much sense now.